moped-editor/src/views/projects/projectView/ProjectStatusBadge.js

```diff
@@ -10,128 +10,129 @@ import HelpOutlineIcon from "@mui/icons-material/HelpOutline";
 
 import { makeStyles } from "@mui/styles";
 import clsx from "clsx";
+import theme from "src/theme";
 
 const defaultIcon = HelpOutlineIcon;
 const defaultLabel = "Unknown";
 
+/**
+ * Font colors
+ */
+const primary = theme?.palette?.text?.primary;
+const white = theme?.palette?.background?.paper;
+
+/**
+ * Background color mapping
+ */
+const backgroundColors = {
+  default: theme?.palette?.grey?.[300],
```

Comment:
i agree that the grey is a little to light on the map. maybe something to worry about later, but i think it'd be fine to make it darker on the map